charts/kube-prometheus-stack/values.yaml

```diff
@@ -1649,17 +1651,18 @@ prometheusOperator:
       enabled: false
       # self-signed root certificate
       rootCert:
-        duration: ""  # default to be 5y
+        duration: "" # default to be 5y
       admissionCert:
-        duration: ""  # default to be 1y
+        duration: "" # default to be 1y
```

Comment:
Add two spaces before #. Example has below,
duration: ""  # default to be 1y

Reply:
Thanks, I fixed my zealous YAML linter.